apiary.apib

```diff
@@ -1785,6 +1785,28 @@ Get a list of available payment terms.
                 + id: `c93ddb52-0af8-47d9-8551-441435be66a7` (string)
                 + Include PaymentTerm
 
+### productCategories.list [GET /productCategories.list]
+
+Get a list of product categories.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + filter (object, optional)
+            + department_id: `080aac72-ff1a-4627-bfe3-146b6eee979c` (string, optional) - The ID of the department you wish to filter on.
+
++ Response 200 (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + data (array)
+            + (object)
+                + id: `2aa4a6a9-9ce8-4851-a9b3-26aea2ea14c4` (string)
+                + name: 'Asian Flowers' (string)
+                + ledgers (array)
+                    + (object)
+                        + department_id: `2aa4a6a9-9ce8-4851-a9b3-26aea2ea14c6` (string)
+                        + ledger_account_number: 70100 (number)
```

Comment:
Can this also have leading zeros of letters in some countries?

Reply:
Tagging @PieterReel for input.

Reply:
:ok: by @luisabritotl on 13/03/2018 09:01:47